projects/touch.fan/index.js

```diff
@@ -1,21 +1,48 @@
-const { nullAddress } = require('../helper/unwrapLPs')
+const { sumTokensExport } = require('../helper/unknownTokens')
+
+const { default: BigNumber } = require("bignumber.js");
+const { sumTokens2, nullAddress } = require('../helper/unwrapLPs')
 
 const fan_contract = "0x9842114F1d9c5286A6b8e23cF0D8142DAb2B3E9b"
+
 const touch_contract = "0xC612eD7a1FC5ED084C967bD71F1e0F0a338Cf816"
+const usdc_address = "0x3355df6d4c9c3035724fd0e3914de96a5a83aaf4"
 const tft_address = "0x14acccd04393f26ba155e5402aa6fddbb8e2254a"
 const zk_sync_swap_pool = "0xB075F49F4Bea5204d22B42E8d442e8E9e3400AF8"
 const stake_tft_contract_address = "0x38a22eFB9eE73EdABdc4A15B17D28b70bA56BDA3"
 const getReserves_abi = "function getReserves() external view  returns (uint,uint)"
 const vester_address = "0x13B8C03990b7d782cB9c3ebF7E2aA5Dc28f6133B"
+const lock_tft_address = "0x9b470d1d40F437cAb76ea8fC80BC3b761e96D44D"
+const lock_lp_address = "0x26854C6A28864D7E84893F45EDf366eE2494B09a"
 
-const { sumTokensExport } = require('../helper/unknownTokens')
+async function getSwapPool({ api }) {
+    const reserves = await api.call({ abi: getReserves_abi, target: zk_sync_swap_pool })
+    const [tft, usdc] = reserves
+    const tftAmout = BigNumber(tft)
+    const usdcAmount = BigNumber(usdc)
+    const tftPrice = usdcAmount.times(10 ** 12).div(tftAmout)
+    return [tftPrice, BigNumber(usdc).times(2), tftAmout]
+}
 
+async function pool2(_t, _, _b, { api }) {
```

Comment:
hi @TouchFan , we count the staked LP as pool2 not entire token amount in the LP

Reply:
thank you

Reply:
I have modified , please review it again